File: POO_course/JS/driver.js
class Driver extends Account {
    constructor(name, document, email, password) {
        super(name, document, email, password);
    }

    printDataDriver = () => {
        console.log("Driver name: ", this.name, ", Driver document: ", this.document, ", Email: ", this.email, ", Password: ", this.password);
    }
}

File: POO_course/Java/Driver.java
class Driver extends Account {
    public Driver(String name, String document, String email, String password) {
        super(name, document, email, password);
    }
    void printDataDriver() {
        System.out.println("Driver Document: " + document + " Driver name: " + name + " Email: " + email + " Password: " + password + " ");
    }
}